jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/GettingStartedOnStartup.kt

```diff
@@ -38,15 +38,15 @@ class GettingStartedOnStartup : StartupActivity {
                 GettingStartedPanel.openPanel(project, firstInstance = true, connectionInitiatedFromExplorer = false)
                 AuthTelemetry.addConnection(
                     project,
-                    source = SourceOfEntry.FIRST_STARTUP.name,
+                    source = SourceOfEntry.FIRST_STARTUP.toString(),
```

Comment:
i'm starting to think we need to centralize the enum -> toString() logic somewhere